src/compiler/crystal/macros/methods.cr

```diff
@@ -927,6 +927,16 @@ module Crystal
         interpret_argless_method(method, args) { @of.try(&.value) || Nop.new }
       when "type"
         interpret_argless_method(method, args) { @name || Nop.new }
+      when "delete"
+        interpret_one_arg_method(method, args) do |key|
+          unless key.is_a? Crystal::NumberLiteral | Crystal::SymbolLiteral | Crystal::StringLiteral | Crystal::BoolLiteral
+            key.raise "argument to HashLiteral#delete must be a number, bool, symbol, or string, not #{key.class_desc}:\n\n#{key}"
+          end
+
+          entries.reject! { |e| e.key == key }
```

Comment:
```suggestion
          entries.reject!(&.key.==(key))
```

Reply:
IMO that's another installment of "not really an improvement" 🤷 The short block syntax is just harder to read.

Reply:
> [...] The short block syntax is just harder to read.

I disagree. For me it's quite the opposite (what's `e`?)

Reply:
entry

Reply:
Also, it's already being used:

https://github.com/crystal-lang/crystal/blob/605ec43fcfa7286eae3e28ad212514bda0d3f3e3/src/compiler/crystal/macros/methods.cr#L913

Reply:
In general I would ignore these style comments. Not because they are wrong, it's just a matter of personal preference. The formatter accepts both so they are both fine. And in that case, I would avoid suggesting such changes because they don't add any value. On the contrary, it adds noise to a PR.

Reply:
Yes it is, so it's up to the OP to use 'em or not. All this discussion is totally pointless and IMO this is what adds the most noise, not the suggestions themselves.

Reply:
The thing is: When you make such a suggestion without any comment it looks very determined and people may just follow your suggestion, not for substatial reason but because you asked them to. And that leads to pointless edits. The code would be accepted either way.
Do we need to declare a style guide to prevent this or can we just agree to avoid purely stylistic change suggestions?

Reply:
Absence of any comment re: the change was a mistake from my side, agreed. I see value in using idiomatic code though, IMO it improves the readability of the code and helps in learning the language.

Reply:
We could remove using operators in such a way. I personally dislike it.